### PR TITLE
FIX: Cross-compile

### PR DESCRIPTION
**Pull request by formych**
Cross-compile your app inside the Docker container

I have tried to run this code， but it did not work. 
So I changed in my way.

### CODE REVIEW

File: golang/content.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Alternatively, you can build for multiple platforms at once:
 $ docker run --rm -it -v "$PWD":/usr/src/myapp -w /usr/src/myapp %%IMAGE%%:1.8 bash
 $ for GOOS in darwin linux; do
 >   for GOARCH in 386 amd64; do
->     go build -v -o myapp-$GOOS-$GOARCH
+>     GOOS=$GOOS GOARCH=$GOARCH go build -v -o myapp-$GOOS-$GOARCH
```
Comment (tianon) [inline golang/content.md]: Oh good catch, these need to be exported.  In the interest of shorter line length and making it more obvious what's happening here, I think I'd prefer to go for an explicit `export`, ala:

```suggestion
>     export GOOS GOARCH
>     go build -v -o myapp-$GOOS-$GOARCH
```

Comment (formych) [reply golang/content.md]: Oh, what you said is right! But i think  this change the global env GOOS and GOARCH, maybe  we should unexport then in the end.

Comment (tianon) [reply golang/content.md]: This is just an example, so I think that's best left as an exercise for the reader.

Comment (formych) [reply golang/content.md]: Okey,  i write as u recommened, maybe i think too more, ^__^

**Comment by tianon** [comment]
Awesome, thanks!

**Comment by tianon** [comment]
I guess the simpler option here would've been to just use `bash -a` (https://www.gnu.org/software/bash/manual/bashref.html#The-Set-Builtin). 😅

**Pull request merged**